deployments/hadoop-yarn/ansible/39-create-user-scripts.yml

```
@@ -125,27 +125,31 @@
             --data "password=${NEW_PASSWORD:?}" \
             ${ZEPPELIN_URL:?}/api/login 
 
-           curl --silent --cookie "${zepcookies:?}" "${ZEPPELIN_URL:?}/api/notebook"| jq -r '.body[] | select(.path | startswith("/Public")) | [.id, .path] | @tsv' |
-             while IFS=$'\t' read -r id path; do
-               curl -L         -H 'Content-Type: application/json'         -d "{'name': '${path/Public Examples/Users/$NEW_USERNAME}' }"        --request POST         --cookie "${zepcookies:?}"    $ZEPPELIN_URL/api/notebook/$id
-             done
+            curl --silent --cookie "${zepcookies:?}" "${ZEPPELIN_URL:?}/api/notebook"| jq -r '.body[] | select(.path | startswith("/Public")) | [.id, .path] | @tsv' |
+            while IFS=$'\t' read -r id path; do
+              curl -L         -H 'Content-Type: application/json'         -d "{'name': '${path/Public Examples/Users/$NEW_USERNAME}' }"        --request POST         --cookie "${zepcookies:?}"    $ZEPPELIN_URL/api/notebook/$id
+            done
 
     
     add_user: |
             #!/bin/bash
-            printf "Create a new Zeppelin user"
-            printf "\n"
-            printf "Username: "
-            read NEW_USERNAME
-            
-            stty -echo
-            printf "Password: "
-            read NEW_PASSWORD
-            stty echo
-                        
-            printf "\n"
-            printf "User role: "
-            read NEW_USER_ROLE
+            if [ "$#" -ne 3 ]
+            then
+                
+                printf "Create a new Zeppelin user"
+                printf "\n"
+                printf "Username: "
+                read NEW_USERNAME
+                
+                stty -echo
+                printf "Password: "
+                read NEW_PASSWORD
+                stty echo
+                            
+                printf "\n"
+                printf "User role: "
+                read NEW_USER_ROLE
+            fi
```

Do we need an `else` here ?
```suggestion
            else
                NEW_USERNAME=$1
                NEW_PASSWORD=$2
                NEW_USER_ROLE=$3
            fi
```